tutorials/LinearAlgebra/LinearAlgebra.ipynb

```diff
@@ -0,0 +1,983 @@
+    "\n",
+    "A **matrix** is a rather simple concept - it's a set of numbers grouped together in a rectangular grid. Here is a $2$ by $2$ matrix:\n",
+    "\n",
+    "$$A =\n",
+    "\\begin{bmatrix} 1 & 2 \\\\ 3 & 4 \\end{bmatrix}$$\n",
+    "\n",
+    "$A_{i,j}$ Refers to the element in row $i$ and column $j$ of matrix $A$. In the above example, $A_{0,1} = 2$.\n",
+    "\n",
+    "An $n \\times m$ matrix will have $n$ rows and $m$ columns, like so:\n",
+    "\n",
+    "$$\\begin{bmatrix}\n",
+    "    x_{0,0} & x_{0,1} & \\dotsb & x_{0,m-1} \\\\\n",
+    "    x_{1,0} & x_{1,1} & \\dotsb & x_{1,m-1} \\\\\n",
+    "    \\vdots  & \\vdots  & \\ddots & \\vdots  \\\\\n",
+    "    x_{n-1,0} & x_{n-1,2} & \\dotsb & x_{n-1,m-1}\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "A $1 \\times 1$ matrix is equivalent to a scalar:\n",
+    "\n",
+    "$$\\begin{bmatrix} 3 \\end{bmatrix} = 3$$\n",
+    "\n",
+    "In quantum computing, we use complex valued matrices: the elements of a matrix can be complex numbers. This, for example, is a valid complex-valued matrix:\n",
+    "\n",
+    "$$\\begin{bmatrix}\n",
+    "    1 & i \\\\\n",
+    "    -2i & 3 + 4i\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "Finally, for the purposes of linear algebra, a **vector** is an $n \\times 1$ matrix. Here, for example, is a $3 \\times 1$ vector:\n",
+    "\n",
+    "$$\\begin{bmatrix} 1 \\\\ 2i \\\\ 3 + 4i \\end{bmatrix}$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Matrix Addition\n",
+    "\n",
+    "The easiest matrix operation to understand is **matrix addition**. Matrix addition works between two matrices of the same size, and adds each number from the first matrix to the number in the same position in the second matrix:\n",
+    "\n",
+    "$$\\begin{bmatrix}\n",
+    "    x_{0,0} & x_{0,1} & \\dotsb & x_{0,m-1} \\\\\n",
+    "    x_{1,0} & x_{1,1} & \\dotsb & x_{1,m-1} \\\\\n",
+    "    \\vdots  & \\vdots  & \\ddots & \\vdots  \\\\\n",
+    "    x_{n-1,0} & x_{n-1,1} & \\dotsb & x_{n-1,m-1}\n",
+    "\\end{bmatrix}\n",
+    "+\n",
+    "\\begin{bmatrix}\n",
+    "    y_{0,0} & y_{0,1} & \\dotsb & y_{0,m-1} \\\\\n",
+    "    y_{1,0} & y_{1,1} & \\dotsb & y_{1,m-1} \\\\\n",
+    "    \\vdots  & \\vdots  & \\ddots & \\vdots  \\\\\n",
+    "    y_{n-1,0} & y_{n-1,1} & \\dotsb & y_{n-1,m-1}\n",
+    "\\end{bmatrix}\n",
+    "=\n",
+    "\\begin{bmatrix}\n",
+    "    x_{0,0} + y_{0,0} & x_{0,1} + y_{0,1} & \\dotsb & x_{0,m-1} + y_{0,m-1} \\\\\n",
+    "    x_{1,0} + y_{1,0} & x_{1,1} + y_{1,1} & \\dotsb & x_{1,m-1} + y_{1,m-1} \\\\\n",
+    "    \\vdots  & \\vdots  & \\ddots & \\vdots  \\\\\n",
+    "    x_{n-1,0} + y_{n-1,0} & x_{n-1,1} + y_{n-1,1} & \\dotsb & x_{n-1,m-1} + y_{n-1,m-1}\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "Matrix addition has the following properties:\n",
+    "\n",
+    "* Commutativity: $A + B = B + A$\n",
+    "* Associativity: $(A + B) + C = A + (B + C)$"
```

Comment:
I wonder if we should mention matrix subtraction somewhere? No need to write out the formulas, something like this: "Similarly, we can subtract one matrix from another: A − B is computed by subtracting elements of B from corresponding elements of A."